public/index.html

```diff
@@ -3,24 +3,29 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
   </head>
   <body>
-    <h1>Hello from Hoodie!</h1>
-
-    <p>You see this because your app has no <code>public/</code> folder.</p>
-    <p>But don't worry, we're here to get you started! </p>
-
-    <h2>Get going with Hoodie</h2>
-
-    <p>1. Your first step should be creating a <code>public</code> folder inside your project folder.</p>
-    <p>2. Next up, just create your <strong>index.html</strong> inside that public directory.</p>
-    <p>3. We're almost here: Paste this into your <strong>index.html</strong> just before your closing tag for the body: <strong> &lt;script src="hoodie/client.js"&gt;&lt;/script&gt;</strong></p>
-    <p>This is necessary for your own javascript files to pick up the hoodie methods.</p>
-    <p>Now handle it just like you would handle your frontend project, hoodie won't get in your way. </p>
-    <p>4. Setup your asset folders and create your .css and .js files.</p>
-    <p>5. You’ll need to restart the Hoodie server so it servers assets from your app’s <code>public</code> folder</p>
-
-    <p>If you want to learn more, check the <a href="http://docs.hood.ie/camp/">work-in-progress documentation</a></p>
-    <p>Happy coding :)</p>
-</body>
+    <h1>Well done, you made it!</h1>
+    <div id="doggy">
+      <img src="low-profile-dog-1.png" />
+    </div>
+    <div class="info">
+      Hoodie is now running a local server.
```

Comment:
I think `<h2>` makes it way too visible perhaps a `<strong>` tag (although nowadays its being phased out a little). 

![screen shot 2016-10-03 at 11 25 50 pm](https://cloud.githubusercontent.com/assets/1141128/19062573/ef197df6-89c0-11e6-8f72-ae0ba8d58f21.png)